### PR TITLE
[Port] TGUI Manifests

### LINKED ISSUE #11095: Security Record UX
- If you add a crime before arresting someone, it will be added again when the prison manager console adds the crime.
- If you are dragging someone, it would be helpful if there was a way to automatically fill out their name.
- Remove the comments section, they are hidden away and missed and it is ambiguous when people set crimes vs set comments.
- When you enter the name on the prisoner management interface, it should show their record somewhere and ask if the crime needs to be added
- Tabify the prisoner management vs the prisoner main interface, rather than have a big scroll menu
- It would be nice if there was a system to request arrest warrants which can have evidence or reasoning attached to it, rather than just setting them to arrest. Less people should come into brig with no crimes attached to their record, causing confusion. CM has a very nice system where you add evidence to security records, and witnesses when you mark someone to arrest.

### PR DESCRIPTION
**Pull request by XeonMations**
## About The Pull Request

TGUI-ifies the security, medical and general manifests.

Ports:
- https://github.com/tgstation/tgstation/pull/83740
- https://github.com/tgstation/tgstation/pull/82105
- https://github.com/tgstation/tgstation/pull/81637
- https://github.com/tgstation/tgstation/pull/82103
- https://github.com/tgstation/tgstation/pull/78528
- https://github.com/tgstation/tgstation/pull/78516
- https://github.com/tgstation/tgstation/pull/73084
- https://github.com/tgstation/tgstation/pull/73666
- https://github.com/tgstation/tgstation/pull/73691
- https://github.com/tgstation/tgstation/pull/75320
- https://github.com/tgstation/tgstation/pull/76760
- https://github.com/tgstation/tgstation/pull/73022
- https://github.com/tgstation/tgstation/pull/73882
- https://github.com/tgstation/tgstation/pull/67508
- https://github.com/tgstation/tgstation/pull/72725

Fixes https://github.com/BeeStation/BeeStation-Hornet/issues/11095

## Why It's Good For The Game

alot of newer code + better performing and nicer looking security, general and medical manifests

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl:
refactor: Refactored the security, medical and general manifests to TGUI.
/:cl:

### CODE REVIEW

**Comment by MarkusLarsson421** [comment]
"day 1" of ??? years. :P

File: code/__DEFINES/jobs.dm [hunk 1]
```diff
@@ -11,6 +11,58 @@
...
+#define JOB_CREW_MEMBER (1<<3)
+/// Whether this job can be joined through the new_player menu.
+#define JOB_NEW_PLAYER_JOINABLE (1<<4)
+/// Whether this job appears in bold in the job menu.
+#define JOB_BOLD_SELECT_TEXT (1<<5)
+/// Reopens this position if we lose the player at roundstart.
+#define JOB_REOPEN_ON_ROUNDSTART_LOSS (1<<6)
+/// If the player with this job can have quirks assigned to him or not. Relevant for new player joinable jobs and roundstart antags.
+#define JOB_ASSIGN_QUIRKS (1<<7)
+/// Whether this job can be an intern.
```
Comment (tontyGH) [inline code/__DEFINES/jobs.dm:31]: We don't have an intern system

File: code/__DEFINES/jobs.dm [hunk 2]
```diff
@@ -11,6 +11,58 @@
...
+#define JOB_CAN_BE_INTERN (1<<8)
```
Comment (tontyGH) [inline code/__DEFINES/jobs.dm:32]: We don't have /tg/'s intern system

Comment (XeonMations) [reply code/__DEFINES/jobs.dm:32]: Not yet.

Comment (tontyGH) [reply code/__DEFINES/jobs.dm:32]: Blows you up with my mind

**Pull request closed without merging**